catalog/ui/search-ui/search-endpoint/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -20,9 +20,17 @@
         <AD description="Disables use of cache." name="Disable Cache" id="cacheDisabled"
             required="false" type="Boolean" default="false"/>
 
-        <AD description="Disables relevance and distance normalization." name="Disable Normalization"
+        <AD description="Disables relevance and distance normalization."
+            name="Disable Normalization"
             id="normalizationDisabled" required="false" type="Boolean" default="false"/>
 
+        <AD description="Controls the maximum number of results returned from a single query. This
```

Comment:
It isn't needed and I was on the fence about adding it to be honest. Will remove.